src/resolvers/Organization/image.ts

```diff
@@ -0,0 +1,13 @@
+import { BASE_URL } from "../../constants";
+import { Organization } from "../../models/Organization";
+import { OrganizationResolvers } from "../../types/generatedGraphQLTypes";
+
+export const image: OrganizationResolvers["image"] = async (parent) => {
+  const org = await Organization.findOne({
```

Comment:
You don't need to fetch organization object again. It's already fetched in the parent resolver. Make use of `parent.image` to get the image.